slm_porosity/heat_transfer.f90

```diff
@@ -39,6 +39,9 @@ MODULE user_case
   REAL (pr) :: scanning_speed
   REAL (pr) :: initial_porosity
   REAL (pr) :: initial_enthalpy
+  REAL (pr) :: dh_step              ! step for spline function maybe temporary
```

Comment:
Let this parameter be called `smoothing_width`.

Reply:
Done